### DISCUSSION TITLE
Gitbook as a corporate website

### DISCUSSION THREAD

**Discussion by kmarshall008** [GitbookIO/community#253] [0. General bugs and questions] [answered]
Are many companies using Gitbook as a corporate website? Can you provide examples? 

Thx, nice product.

**Comment by aikoii** [comment, marked as answer]
Hi @kmarshall008,

You can head to [GitBook Library](https://www.gitbook.com/gitbook-library) to view how our customers use GitBook! 🙂

Hope this helps 👋.
